webapp/src/component/navigation/NavigationWrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import grey from '@material-ui/core/colors/grey';
+import { Box, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  container: {
```

Comment:
display, alignItems could be provided as props of Box component

Reply:
Fixed